nvflare/fuel/utils/log_utils.py

```diff
@@ -262,6 +262,42 @@ def apply_log_config(dict_config, dir_path: str = "", file_prefix: str = ""):
     logging.config.dictConfig(dict_config)
 
 
+def dynamic_log_config(config: str, workspace: Workspace, job_id: str = None):
+    # Dynamically configure log given a config (filepath, levelname, levelnumber, 'reload'), apply the config to the proper locations.
+    if not isinstance(config, str):
+        raise ValueError(
+            f"Unsupported config type. Expect config to be string filepath, levelname, levelnumber, or 'reload' but got {type(config)}"
+        )
+
+    if config == "reload":
+        config = workspace.get_log_config_file_path()
```

Comment:
Do we need to "apply_log_config" for reload?

Reply:
it will get called below since config will be set to the log config file path